packages/jest-config/src/normalize.js

```diff
@@ -354,6 +354,9 @@ function normalize(config: InitialConfig, argv: Object = {}) {
       case 'persistModuleRegistryBetweenSpecs':
       case 'preset':
       case 'replname':
+      case 'reporters':
+        value = _replaceRootDirTags(config.rootDir, config[key]);
```

Comment:
The previous PR #2115 adds a validation here in order to check the type of the options passed. But after `jest-validate` that doesn't seem to be necessary.